apps/server/src/modules/tldraw/controller/api-test/tldraw.controller.401.api.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { INestApplication } from '@nestjs/common';
+import { EntityManager } from '@mikro-orm/mongodb';
+import { courseFactory, TestXApiKeyClient, UserAndAccountTestFactory } from '@shared/testing';
+import { Test, TestingModule } from '@nestjs/testing';
+import { ServerTestModule } from '@modules/server';
+import { Logger } from '@src/core/logger';
+import { TldrawService } from '../../service';
+import { TldrawController } from '..';
+import { TldrawRepo } from '../../repo';
+import { tldrawEntityFactory } from '../../testing';
+
+const baseRouteName = '/tldraw-document';
+describe('tldraw controller (api)', () => {
+	let app: INestApplication;
+	let em: EntityManager;
+	let testXApiKeyClient: TestXApiKeyClient;
+	const API_KEY = '7ccd4e11-c6f6-48b0-81eb-cccf7922e7a4';
+
+	beforeAll(async () => {
+		const module: TestingModule = await Test.createTestingModule({
+			imports: [ServerTestModule],
+			controllers: [TldrawController],
+			providers: [Logger, TldrawService, TldrawRepo],
+		}).compile();
```

Comment:
so, the difference is that you don't overwrite the AuthGuard as you do in tldraw.controller.api.spec.ts
I would put that in the description... The user is technically valid, but the authentication is not allowing

Reply:
we don't provide token in request header that is why authentication is failing

Reply:
exactly, but that is something hard to read here. You do seem to call the Client with an API_KEY, which makes it confusing for anyone reading the code. Hence, some hints in the description of the test would make it easier to understand this point